language-plutus-core/test/types/verifyIdentity.plc

```diff
@@ -1,30 +1,32 @@
 (program 0.1.0
-  (lam pubkey [(con bytestring) (con 256)]
-    (lam signed [(con bytestring) (con 32)]
-      [ 
-        {
-          (abs a (type)
-            (lam b (all a (type) (fun a (fun a a)))
-              (lam t (fun (all a (type) (fun a a)) a)
-                (lam f (fun (all a (type) (fun a a)) a)
-                  [
-                    [ { b (fun (all a (type) (fun a a)) a) } t f ]
-                    (abs a (type) (lam x a x))
-                  ]
+  (lam txhash [(con bytestring) (con 32)]
+    (lam pubkey [(con bytestring) (con 256)]
```

Comment:
I just did a random change here in order to make this thing type check. Is it fine?

Reply:
Seems fine.